src/main/java/fi/foyt/foursquare/api/entities/List.java

```diff
@@ -44,6 +44,34 @@
   private String createdAt;
   private String updatedAt;
   private String entities;
-  private Boolean isPublic; public void setPublic(Boolean isPublic) { this.isPublic = isPublic; }
+  private Boolean isPublic;
+  public void setPublic(Boolean isPublic) { this.isPublic = isPublic; }
```

Comment:
if we're modifying this line i'd rather pull this function out into the rest of the class with appropriate formatting:
```
public void setPublic(Boolean isPublic) {
    this.isPublic = isPublic;
}
```